src/_assets/stylesheets/elements/_inputs.scss

```diff
@@ -0,0 +1,3 @@
+select {
+  padding-right: 4rem;
+}
```

Comment:
Solving this particular UI bug. 👇 

<img width="369" alt="screen shot 2017-05-15 at 4 09 37 pm" src="https://cloud.githubusercontent.com/assets/27780860/26077099/ec1ada0a-3988-11e7-8fdc-a95b2356b613.png">